deployment/eks/cao-config.tf

```diff
@@ -0,0 +1,36 @@
+locals {
+
+  rdb_endpoint   = var.use_rds_and_elastic_cache ? aws_db_instance.mysql_instance[0].endpoint : kubernetes_service.mysql[0].metadata.0.name
+  cache_endpoint = var.use_rds_and_elastic_cache ? aws_elasticache_cluster.redis_instance[0].cache_nodes.0.address : kubernetes_service.redis[0].metadata.0.name
```

Comment:
Please rename `cache_endpoint` to `redis_endpoint`.

Reply:
Done.